includes/class-migration.php

```diff
@@ -167,6 +167,9 @@ public static function maybe_migrate() {
 		if ( \version_compare( $version_from_db, '4.7.2', '<' ) ) {
 			self::migrate_to_4_7_2();
 		}
+		if ( \version_compare( $version_from_db, 'unreleased', '<' ) ) {
```

Comment:
Why `unreleased`? Are you just trying to ensure that we always flush rewrites in any and every upgrade routine?

Reply:
The release script will release it with the next version that gets released. It's supposed to help avoid what happened in 4.7.0 where an upgrade script was merged after the targeted release had already gone out (#1161).

Reply:
Aha! Very nice. Should be good then.